config/travis/run_python3.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
 #
 # Script to run Python 3 tests on Travis-CI.
+#
+# This file is generated by l2tdevtools update-dependencies.py, any dependency
+# related changes should be made in dependencies.ini.
 
 # Exit on error.
 set -e;
 
 python3 ./run_tests.py
 
 python3 ./setup.py build
+
 python3 ./setup.py sdist
+
 python3 ./setup.py bdist
+
+# python3 ./setup.py install
```

Comment:
Can you add a comment to explain the reason why this one is disabled?

Reply:
Let me test enabling this.